scikeras/utils/transformers.py

```diff
@@ -368,7 +368,7 @@ def inverse_transform(self, y: np.ndarray) -> np.ndarray:
             Keras Model predictions cast to the dtype and shape of the input
             targets.
         """
-        if y.dtype == np.float32:
+        if y.dtype.name == "float32":
```

Comment:
:+1: this helps make SciKeras work with other array libraries (CuPy, Dask, XArray, ...).

Reply:
Great, thanks for the tip! I think I got most of them here since they were all in the file we're editing for this PR anyway, but I also opened https://github.com/adriangb/scikeras/issues/158 to try to track down any stray occurrences of this pattern in other files.